x11-wm/notion/notion-9999.ebuild

```diff
@@ -30,16 +30,16 @@ DEPEND="${RDEPEND}
 
 src_prepare() {
 	sed -e "/^CFLAGS=/s:=:+=:" \
-		-e "/^CFLAGS/{s:-Os:: ; s:-g::}" \
-		-e "/^LDFLAGS=/{s:=:+=: ; s:-Wl,--as-needed::}" \
-		-e "/^CC=/s:=:?=:" \
-		-e "/^PREFIX/s:=.*$:= \${ROOT}usr:" \
-		-e "/^ETCDIR/s:=.*$:= \${ROOT}etc/notion:" \
-		-e "/^LIBDIR/s:=.*:= \$(PREFIX)/$(get_libdir):" \
-		-e "/^DOCDIR/s:=.*:= \$(PREFIX)/share/doc/${PF}:" \
-		-e "/^LUA_DIR/s:=.*$:= \$(PREFIX)/usr:" \
-		-e "/^VARDIR/s:=.*$:= \${ROOT}var/cache/${PN}:" \
-		-e "s:^\(X11_PREFIX=\).*:\1\$(PREFIX)/usr:" \
+                -e "/^CFLAGS/{s:-Os:: ; s:-g::}" \
```

Comment:
Please set your editor to always use tabs to indent ebuilds. I'm going to fix it this time when committing.